src/psyclone/psyir/symbols/symbol_table.py

```diff
@@ -1636,22 +1777,32 @@ def rename_symbol(self, symbol, name, dry_run=False):
         # Re-insert modified symbol
         self.add(symbol)
 
-    def has_wildcard_imports(self):
+    def wildcard_imports(self, collect_all=False):
```

Comment:
I feel the parameter default value is unexpected here, if a user does `symtab.wildcard_imports()` without reading the docs it would expect to get all of them. Is this just a performance optimisation for the case that we want to check if there are any? Going up (is just one parent ) and lookups are not that expensive, I think I would remove the argument altogether.

Reply:
Agreed. It's gone now. Should I revert to a property or leave it as an explicit call?